risc0/zkvm/Cargo.toml

```diff
@@ -49,8 +50,20 @@ num-traits = { version = "0.2", default-features = false }
 prost = { version = "0.11", optional = true }
 rand = { version = "0.8", optional = true }
 rayon = { version = "1.5", optional = true }
+reqwest = { version = "0.11.16", features = [ "blocking", "json" ] }
 rrs-lib = { version = "0.1", optional = true }
 sha2 = { version = "0.10", optional = true }
+serde_json = "1.0.96"
+serde_with = { version = "2.0.0", features = ["time_0_3"] }
+time = "0.3.11"
+utoipa = "3.3.0"
+sqlx = { version = "0.6.2", features = [
```

Comment:
Do we need a database driver? The types should only need serde traits. This brings in a lot of code to the tree.

Reply:
shoot! I had unintentionally left these in there from when I lifted the struct definitions! Thanks for catching it!